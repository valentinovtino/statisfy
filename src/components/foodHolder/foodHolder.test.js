import React from 'react';
import {shallow } from 'enzyme';
import FoodHolder from './foodHolder';

describe('FoodHolder', () => {

  let foodHolder;
  let props = {
    allState: {},
    food: []
  };

  beforeEach(() => {
    foodHolder = shallow(<FoodHolder {...props} />);
  });

  it('matches snapshot', () => {
    expect(foodHolder).toMatchSnapshot();
  });
});