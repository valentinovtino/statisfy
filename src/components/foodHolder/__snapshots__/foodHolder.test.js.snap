// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodHolder matches snapshot 1`] = `undefined`;
